target/linux/realtek/dts/rtl9303_tplink_tl-st5008.dts

```diff
@@ -0,0 +1,572 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rtl930x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/thermal/thermal.h>
+
+/ {
+	compatible = "tplink,tl-st5008", "realtek,rtl930x-soc";
+	model = "TP-Link TL-ST5008";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x00000000 0x10000000>, /* first 256 MiB */
+		      <0x20000000 0x10000000>; /* remaining 256 MiB */
+	};
+
+	aliases {
+		led-boot = &led_sys_red;
+		led-failsafe = &led_master_yellow;
+		led-running = &led_sys_green;
+		led-upgrade = &led_stack;
+		label-mac-device = &ethernet0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		button-mode {
+			label = "mode";
+			gpios = <&gpio0 4 GPIO_ACTIVE_LOW>;
+			debounce-interval = <50>;
+			linux,code = <BTN_0>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_master_green: led-master-green {
```

Comment:
Lot of these labels are not actually used. This is one example but there are more like the `*_trip1` ones. You should remove them in this case